maplibre/src/error.rs

```diff
@@ -3,6 +3,7 @@
 use lyon::tessellation::TessellationError;
 use std::sync::mpsc::SendError;
 
+/// Multiple types of map rendering errors
```

Comment:
```suggestion
/// Enumeration of errors which can happen during the operation of the library.
```

It's not only rendering errors.